src/components/App/index.tsx

```diff
@@ -17,32 +17,40 @@ const App = props => {
 
   useEffect(() => {
     httpGet(
-      constants.API_HEALTHCHECK_HELLO,
+      constants.API_MIRROR_HEALTHCHECK_HELLO,
       null,
-      process.env.REACT_APP_ONEAUTH_API_URL
-    )
-      .then(response => {
-        if (response.status === 200) {
-          setHealthCheck({
-            ...healthCheck,
-            isVerified: true,
-            allowAccess: true,
+      process.env.REACT_APP_GRAPHQL_URL
```

Comment:
you cannot use REACT_APP_GRAPHQL_URL. This points to baseurl/graphql. Healthcheck URL is baseurl/hello. So, create a new env variable to hold REACT_APP_API_HEALTHCHECK_URL